Send announcements example

vicrojo opened the pull request:
I've added this code sample to show how to use the sendAnnouncement operation using the AWS JavaScript SDK. It uses text messages, but can be extended to use SSML and / or Audio. For the rooms, I used the Key RoomName, but it can be switched to using the Room ARN, Profile ARN, or the Profile Name

*Issue #, if available: N/A

*Description of changes: This is an addition of a sample code to send announcements in A4B


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

vicrojo committed:
Send announcements example

I've added this code sample to show how to use the sendAnnouncement operation using the AWS JavaScript SDK. It uses text messages, but can be extended to use SSML and / or Audio. For the rooms, I used the Key RoomName, but it can be switched to using the Room ARN, Profile ARN, or the Profile Name

podlogar reviewed (commented):
In the readme, 

1. Please be specific of Alexa for Business permissions. i.e Do they need Full Permissions?
2. Instead of saying "It uses text messages" , I would recommend: "The script uses passed in string messages,"   since saying text messages sounds like you are using SMS.
3. Please indicate if string value that is passed to function sendAnnouncement is case sensitive or not.

3. Recommend in the future you also add a way to retrieve room ARN to send instead or room name.

vicrojo committed:
Code Update and README.md text update

Added support for RoomARN and added JSDOC tags to document the code
Updated the README.md text

vicrojo committed:
Updated README.md

Updated the permissions required to run the script

podlogar reviewed (approved):
Looks good!
